.env.example

```diff
@@ -23,3 +23,7 @@ host=
 SENDGRID_API_KEY=
 SENDGRID_EMAIL=
 SENDGRID_USERNAME=
+
+# facebook client details
+client_id=
```

Comment:
@fayaz07 what happens when we try to run the app with these empty ? will server crash ?

may be we need checks in the app

Reply:
Yeah server crashes actually